regtests/ww3_tp2.22/input/namelists_2-D.nml

```diff
@@ -0,0 +1,4 @@
+&PRO2 DTIME = 345600. /
```

Comment:
In this and the next name list file I don't see: 
 &LMPN LMPENABLED = T, SDTAIL = T, HSLMODE = 0 /

Reply:
Also I don't see a ww3_grid.nml file so I don't think this file is ever even used.   Since there are added values here, it might be worth making a test of nml since that's the way we're going.  At minimum an issue to test that should be made.

Reply:
@JessicaMeixner-NOAA @alperaltuntas  
Regarding inp vs. nml files, since we have not switch over to nml yet, and not all the nml have been tested, I removed those scripts.  My idea was to keep the PR narrowly focused on the 3 issues it resolves and not branch out to testing the nml files.  I will create an issue for the nml file functionality though.

Reply:
sounds good

Reply:
An issue (#1111) has been added to address this comment.